scripts/release.sh

```diff
@@ -129,11 +129,23 @@ function check_tag_correct() {
 
 # build_release builds the actual release binaries.
 #   arguments: <version-tag> <build-system(s)> <build-tags> <ldflags>
+#              <go-version>
 function build_release() {
   local tag=$1
   local sys=$2
   local buildtags=$3
   local ldflags=$4
+  local goversion=$5
+
+  # Check if the active Go version matches the specified Go version.
+  active_go_version=$(go version | awk '{print $3}' | sed 's/go//')
+  if [ "$active_go_version" != "$goversion" ]; then
+    echo "Error: active Go version ($active_go_version) does not match \
+required Go version ($goversion)."
+    exit 1
+  fi
+
+  echo "Building release for tag $tag with Go version $goversion"
```

Comment:
```suggestion
  local minimum_required_go_version=$5

  # Check if the active Go version matches the specified Go version.
  active_go_version=$(go version | awk '{print $3}' | sed 's/go//')
  if [ "$active_go_version" != "$minimum_required_go_version" ]; then
    echo "Error: active Go version ($active_go_version) does not match \
required Go version ($goversion)."
    exit 1
  fi

  echo "Building release for tag $tag with Go version $minimum_required_go_version"
```

Renaming might convey the purpose of the variable more cogently

Reply:
Would `required_go_version` be sufficient?